hub3/ead/mets.go

```diff
@@ -137,7 +137,7 @@ func updateFileInfo(files map[string]*eadpb.File, fg []*CfileGrp, fa *eadpb.Find
 		file.MimeType = metsFile.AttrMIMETYPE
 		fa.GetMimeTypes()[file.MimeType]++
 
-		size, err := strconv.ParseInt(metsFile.AttrSIZE, 10, 32)
+		size, err := strconv.ParseInt(metsFile.AttrSIZE, 10, 64)
```

Comment:
@kiivihal @Bram74  Ik kan niet op de warning hieronder (regel 145) een comment zetten want het is geen diff dus doe ik het via hier. Weten een van jullie toevallig waar deze protobuf bestand allemaal gebruikt wordt?

Reply:
There were some issue with PHP or Javascript protobuf clients having difficulty with int64 but I can't remember exactly. 

I think in this case it is better to change it to int64 in the protobuf file.

Reply:
@kiivihal Indeed but changing it in the protobuf file means we need to regenerate it for every client that uses it right? I'm trying to figure out which clients are using it.